tools/debug/debugsymbolviewer.py

```diff
@@ -161,7 +161,7 @@ def parse_file(log_file):
 					if (is_text_address(hex(stack_addr))):
 						#If yes, print it's corresponding symbol
                                                 stack_addr = stack_addr - int(offset,16)
-						print_symbol(stack_addr)
+                                                print_symbol(stack_addr)
```

Comment:
Except those two lines, you used tabs for indentation. I don't know this change is necessary.